Tests/UnitTests/AdyenTheme/AdyenThemeTests.swift

```diff
@@ -95,5 +95,174 @@ final class AdyenThemeTests: XCTestCase {
         // Then
         XCTAssertEqual(updatedTheme.toggleStyle.cornerRadius, cornerRadius)
     }
-
+    
+    func test_textfieldMethod_shouldUpdateBackgroundColor() {
+        // Given
+        let theme = AdyenTheme()
+        
+        var newTextFieldStyle = AdyenTextFieldStyle()
+        newTextFieldStyle.backgroundColor = AdyenColorScheme.default.primary
+        
+        // When
+        let updatedTheme = theme.textfield(newTextFieldStyle)
+        
+        // Then
+        XCTAssertEqual(updatedTheme.textFieldStyle.backgroundColor, AdyenColorScheme.default.primary)
+    }
+    
+    func test_textfieldMethod_shouldUpdateTextColor() {
+        // Given
+        let theme = AdyenTheme()
+        
+        var newTextFieldStyle = AdyenTextFieldStyle()
+        newTextFieldStyle.textColor = AdyenColorScheme.default.textOnPrimary
+        
+        // When
+        let updatedTheme = theme.textfield(newTextFieldStyle)
+        
+        // Then
+        XCTAssertEqual(updatedTheme.textFieldStyle.textColor, AdyenColorScheme.default.textOnPrimary)
+    }
+    
+    func test_textfieldMethod_shouldUpdateActiveColor() {
+        // Given
+        let theme = AdyenTheme()
+        
+        var newTextFieldStyle = AdyenTextFieldStyle()
+        newTextFieldStyle.activeColor = AdyenColorScheme.default.highlight
+        
+        // When
+        let updatedTheme = theme.textfield(newTextFieldStyle)
+        
+        // Then
+        XCTAssertEqual(updatedTheme.textFieldStyle.activeColor, AdyenColorScheme.default.highlight)
```

Comment:
Are we ever checking if calling `theme.textfield(newTextFieldStyle)` or any other similar updater preserves values that were set before?
For example, we could check if other theme styles are safe after we update textField style.